app/templates/coco-home-view.pug

```diff
@@ -48,6 +48,8 @@ block content
               p
                 if me.isTeacher()
                   a.btn.btn-lg.btn-primary.my-classes-btn(href="/teachers/classes", data-event-action="Homepage Click My Classes CTA", data-i18n="new_home.go_to_my_classes")
+                  p
+                  a.btn.btn-lg.btn-primary.request-quote(href="/teachers/quote" data-event-action="Homepage Click Request A Quote CTA", data-i18n="new_home.request_quote")
```

Comment:
we have moved to using Modal for licenses now instead of that quote page when you see the other page /home-beta.

You should open this modal on click of Request Quote: https://github.com/codecombat/codecombat/blob/master/app/components/common/ModalGetLicenses.vue

Since it's a Vue modal which you will need to open in Backbone, you could do it like we did for this modal: https://github.com/codecombat/codecombat/blob/master/app/views/core/ShareWithTeachersModal.js